pages/overview.js

```diff
@@ -1,3 +1,70 @@
-export default function OverviewPage() {
-  return <h1>Overview page</h1>;
+import { StyledSection } from "./locationdetail/[id]";
```

Comment:
I think if this is a styled component that you want to use in more than one place. i.e. 

`pages/overview.js` and `pages/locationdetail/[id]`

then i think it is better if we move it to the components folder.